test/parallel/test-http-highwatermark.js

```diff
@@ -14,7 +14,7 @@ let requestReceived = 0;
 const server = http.createServer(function(req, res) {
   const id = ++requestReceived;
   const enoughToDrain = req.connection.writableHighWaterMark;
-  const body = 'x'.repeat(enoughToDrain * 100);
+  const body = 'x'.repeat(enoughToDrain * 200);
```

Comment:
This is the single test that is failing. I saw in its history that it was fixed as a flaky in the past. It seems to fails again and I don't see that it's related to my changes.